.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,62 @@
+name: Bug report
+description: Report an issue you encountered when playing OpenRCT2.
+labels: 
+- bug
+
+body:
+- type: markdown
+  attributes:
+    value: Before filing this bug, [please check if it hasn't already been reported](https://github.com/OpenRCT2/OpenRCT2/issues?q=is%3Aissue+is%3Aopen+label%3Abug). If it has, please add useful information to the existing issue instead.
+
+- type: input
+  attributes:
+    label: Operating System
+    placeholder: Windows 10, 64-bit
+  validations:
+    required: true
+
+- type: input
+  attributes:
+    label: OpenRCT2 build
+    placeholder: OpenRCT2, v0.3.4.1 (d76611e on develop) provided by GitHub
+    description: You can copy the version information from the About window.
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: Describe the issue
+    description: Please describe the issue you are experiencing here.
+  validations:
+    required: true
+
+- type: dropdown
+  attributes:
+    label: Area(s) with issue?
+    description: What things had an issue? Check all that apply.
+    multiple: true
+    options:
+      - The bug also exists in RCT2 (vanilla)
+      - This bug is specific to multiplayer
+  validations:
+    required: false
+
+- type: textarea
+  attributes:
+    label: Steps to reproduce
+    description: We highly suggest including a screenshots and a bug report log (System tray->Report bug).
+    placeholder: Tell us the steps required to trigger your bug.
```

Comment:
This doesn't show up unless you erase the numbers (just commenting the obvious to see if it was intended)

Reply:
Not intended, the placeholder was added later, but I think it's fine as is.